### DISCUSSION TITLE
Clicking on the mouse scroll as a way to open records in a new tab.

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#20330] [Feature Requests]
### Summary

Currently, it is not possible to compare cards, we click on one (view data), and then we have to find the next record and click on it to view the data of the next card. However, if you clicked on the mouse scroll (it can be anything else, but this is probably the standard solution), the tab of this record with the cursor on it would open in a new tab, which would allow for better tab management.

### Motivation

I often have to copy the entire link in a new browser tab to have the data record tab open and then I look for the next data record.

### Detailed Design

After hovering the cursor over a specific record by clicking the scroll button, a new browser tab should open.

### Requirements List

Nothing else than when you hover the cursor over a specific record by clicking the scroll button, a new browser tab should open.

### Drawbacks

A simple solution that will allow you to better manage your cards.

### Alternatives

There isn't

### Adoption Strategy

There isn't braking change, just improvement.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
